tests/unit/test_shell.py

```diff
@@ -10,13 +10,18 @@
 # on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
+import os
```

Comment:
Left this in so the unit test can run successfully with the driver as a dependency in Qldbshell, please advise whether to remove this or not.

Reply:
This show stay since we have kept the driver as submodule in the last minor version update.